src/course/handbook/tech-for-better.md

```diff
@@ -0,0 +1,73 @@
+
+Software developers review all Tech for Better Product ideas as a group and shortlist by confirming their individual preferences.
+
+#### [Week 9](../../syllabus/projects/in-house-build-2/schedule/) - [Analysis Workshop](https://fac-slides.netlify.app/slides/analysis/#0)
+
+Following the shorlisting and selection of products in weeks 8, Product Owners work with their assigned Product Team (software developers) to take a closer look at their product idea.
+
+Together they plan the main path through their application - considering how users will find their service, how users will learn what the service offers and how to use it, as well as what steps users will take when using the app.
+
+The PO and Product Team also think about building a product which will respond to the user needs, and consider the intended and unintended consequences of their product.
+
+#### [Week 10](../../syllabus/projects/tfb-design/schedule/) - TfB Design Week
+
+During the first half of TfB Design Week, the Product Team (software developers) work alongside their own Product Owner(PO) to define their MVP. Sessions include:
+
+**[Definition Workshop](https://fac-slides.netlify.app/slides/definition/#0):** Product Teams and PO search for inspiration from other sources before generating their own ideas and creating low fidelity wireframes.
+
+**[Wireframing](https://www.figma.com/blog/how-to-wireframe/#types-of-wireframes):** Product Teams work together to create a high fildelity prototype of their app, refining it following feedback from there PO and end users.
+
+**[User Testing](https://fac-slides.netlify.app/slides/ut-synthesis/#0):** Product Teams conduct user testing with real users scheduled by their PO, using their high fidelity protoype.
+
+During the second part of the week, the Product Teams turn their attention to the technical apsects of their product. Sessions include:
+
+**[Code Planning](https://fac-slides.netlify.app/slides/code-planning/#0):** This in two parts. During Service Blueprinting the Product Teams identify points of interation between users and their app, and document what should happen in the back end. During Teck Stacks, the Product Teams then identify potential technical options for their project's requirements.
+
+**[Technical Spikes](../../syllabus/projects/TFB-design/spikes/):** Product Teams read and explore documentation on technologies that might be used for their project, or to find a definitive answer for some of the "How might we" questions that came up during the Definition Workshop.
+
+#### [Week 11](../../syllabus/projects/tfb-build-1/schedule/) / [Week 12](../../syllabus/projects/tfb-build-2/schedule/) - TfB Build Week 1 / TfB Build Week 2
+
+Product Teams build their MVP across two build weeks, in partnership with their Product Owners. Each Product Team is also supported through check-in calls and code reviews with volunteer Project Mentors from apprentice employers and FAC alumni.
```

Comment:
```suggestion
Build your MVP across two build sprints, in partnership with your Product Owners. You'll have support and code reviews from [Project Mentors](/course/handbook/mentor-roles/).
```

Can you double check the link here before merging? I forget if we need a `.` before the first `/`